circleci/integration-test

```diff
@@ -40,6 +43,9 @@ USER=$CIRCLE_PROJECT_USERNAME
 : ${CIRCLE_BUILD_NUM?"Missing required env var"}
 BUILD_NUM=$CIRCLE_BUILD_NUM
 
+: ${CIRCLE_TEST_REPORTS?"Missing required env var"}
+: ${CIRCLE_ARTIFACTS?"Missing required env var"}
```

Comment:
Are these things already set in existing repos that use integration-test?

Reply:
Yeah they should be in every CircleCi config at Clever, every app created by `ark init` has CircleCI configs that define the two env vars